.github/workflows/nodejs-publish.yaml

```diff
@@ -27,3 +27,13 @@ jobs:
           NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
           # npm uses the NODE_AUTH_TOKEN environment variable
         run: npm run publish:next
+      - name: Make release commit
+        run: |
+          git config --local user.email "action@github.com"
```

Comment:
I don't think it should prevent the merge of this PR.
But I'm just noticing we would probably need to check whether this user name and email is compatible with Eclipse committer validation. This will be important when this code would be finally moved to an eclipse repository.
@benoitf wdyt ?